interpreter/cling/lib/Interpreter/IncrementalJIT.cpp

```diff
@@ -207,7 +207,8 @@ class Azog: public RTDyldMemoryManager {
 
   void deregisterEHFrames() override {
 #ifdef LLVM_ON_WIN32
-    platform::DeRegisterEHFrames(Addr, Size);
+    // platform::DeRegisterEHFrames(Addr, Size);
```

Comment:
I believe RTDyldMemoryManager has args to a static helper function (for Unix) that the instance method (which is without args call too).

I've pushed the relevant changes for Win64 exceptions here: https://github.com/root-project/cling/pull/164, and hid the code behind a **CLING_WIN_SEH_EXCEPTIONS** macro until you are ready to turn it on.